etip/trackers/management/commands/compare_with_exodus.py

```diff
@@ -0,0 +1,99 @@
+from django.core.management.base import BaseCommand, CommandError
+import requests
+
+from trackers.models import Tracker
+
+EXODUS_API_BASE_URL = 'https://reports.exodus-privacy.eu.org'
+EXODUS_API_BASE_PATH = '/api/trackers'
+FIELDS_TO_LOOKUP = [
+    'name',
+    'code_signature',
+    'network_signature',
+    'website'
+]
+LOOKUP_IS_EXACT = 'EXACT_MATCH'
+LOOKUP_IS_DIFFERENT = 'DIFFERENT'
+LOOKUP_MULTIPLE = 'MULTIPLE_IN_ETIP'
+LOOKUP_NOT_FOUND = 'NOT_FOUND_IN_ETIP'
+
+
+class Command(BaseCommand):
+    help = 'Compare trackers stored in Exodus with trackers from ETIP DB'
+
+    def handle(self, *args, **options):
+        exodus_trackers = self.get_all_from_exodus()
+        self.stdout.write(
```

Comment:
I would add the log line in the get_all_from_exodus function for better readability of the "handle" function.

Reply:
I don't agree; The `get_all_from_exodus` is my service, while logging to stdout is rendering.
This way of rendering is specific to building an admin command and would not be relevant if building a web interface (while calling the api will always be). I know this separation of concerns is not enforced in all the script, but I would keep it where it's right :D
WDYT ?

Reply:
Fair enough!